pkg/executor/connector.go

```diff
@@ -22,6 +23,31 @@ type Connector struct {
 	logs                  Logs
 }
 
+// SubstituteProxyCommand updates variables with values associated with the target host.
+// SSH ProxyCommand can use placeholders such as %h, %p, and %r (%r - username), they have to be replaced with the actual values.
+func SubstituteProxyCommand(username, address string, proxyCommand []string) ([]string, error) {
```

Comment:
Is this one supposed to be public? Unless I missed something, I don't see why the end user would hit it directly.